dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/DataRequestUpdaterTest.kt

```diff
@@ -83,4 +86,108 @@ class DataRequestUpdaterTest {
             Assertions.assertEquals(RequestStatus.open, retrievedDataRequest.requestStatus)
         }
     }
+
+    @Test
+    fun `patch your own answered data request as a premiumUser to closed`() {
+        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.PremiumUser)
+        val stringThatMatchesThePermIdRegex = System.currentTimeMillis().toString()
+        val singleDataRequest = SingleDataRequest(
+            companyIdentifier = stringThatMatchesThePermIdRegex,
+            dataType = SingleDataRequest.DataType.lksg,
+            reportingPeriods = setOf("2022"),
+        )
+
+        val storedDataRequest = requestControllerApi.postSingleDataRequest(singleDataRequest).first()
+        val dataRequestId = UUID.fromString(storedDataRequest.dataRequestId)
+        Assertions.assertEquals(RequestStatus.open, storedDataRequest.requestStatus)
+
+        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.Admin)
+
+        val answeredDataRequest = requestControllerApi.patchDataRequestStatus(dataRequestId, RequestStatus.answered)
+        Assertions.assertEquals(RequestStatus.answered, answeredDataRequest.requestStatus)
+
+        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.PremiumUser)
+
+        val closedDataRequest = requestControllerApi.patchDataRequestStatus(dataRequestId, RequestStatus.closed)
+        Assertions.assertEquals(RequestStatus.closed, closedDataRequest.requestStatus)
+    }
+
+    @Test
+    fun `patch a non owned answered data request as a premiumUser and assert that it is forbidden`() {
```

Comment:
maybe insert a comma after non-owned, so that its more understandable that the data request is non-owned and answered.

Reply:
done!